packages/account/src/Components/poi/idv-document-submit/__tests__/idv-document-submit.spec.tsx

```diff
@@ -67,12 +74,14 @@ describe('<IdvDocumentSubmit/>', () => {
             },
         },
         is_from_external: false,
-        account_settings: {},
-        getChangeableFields: jest.fn(() => []),
     };
 
     it('should render IdvDocumentSubmit component', () => {
-        render(<IdvDocumentSubmit {...mock_props} />);
+        render(
```

Comment:
**NIT**: You could create a function that returns `IdvDocumentSubmit` wrapped with `StoreProvider`